json_logic/__init__.py

```diff
@@ -75,8 +77,19 @@ def to_numeric(arg):
             return int(arg)
     return arg
 
+
+def sum_dates(*args):
+    # Since sum() converts to ints or floats, in the case of dates the normal + operator is needed
+    total = to_numeric(args[0])
+    for arg in args[1:]:
+        total += to_numeric(arg)
```

Comment:
does it still make sense to use `to_numeric` then?

Reply:
No! I should remove it